src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -184,9 +186,6 @@ function getFilterDisplayTitle(filters: Partial<SearchAdvancedFiltersForm>, fiel
         return filters[fieldName];
     }
 
-    // Todo Once all Advanced filters are implemented this line can be cleaned up. See: https://github.com/Expensify/App/issues/45026
-    // @ts-expect-error this property access is temporarily an error, because not every SYNTAX_FILTER_KEYS is handled by form.
-    // When all filters are updated here: src/types/form/SearchAdvancedFiltersForm.ts this line comment + type cast can be removed.
     const filterValue = filters[fieldName] as string;
```

Comment:
So in the end can we remove `as string` or not? 
and if not can we do anything better than casting this value with `as`?